vsintegration/src/FSharp.Editor/DocComments/XMLDocumentation.fs

```diff
@@ -217,6 +218,23 @@ module internal XmlDocumentation =
                 EnsureHardLine collector
                 WriteElement collector el
 
+            match Seq.tryHead (doc.Descendants(XName.op_Implicit "remarks")) with
+            | None -> ()
+            | Some el ->
+                AppendHardLine collector
+                AppendOnNewLine collector (SR.RemarksHeader())
+                AppendHardLine collector
+                WriteElement collector el
+
+            match Seq.tryHead (doc.Descendants(XName.op_Implicit "returns")) with
+            | None -> ()
+            | Some el ->
+                AppendHardLine collector
+                AppendOnNewLine collector (SR.ReturnsHeader())
+                AppendHardLine collector
+                collector.Add(tagSpace "    ")
+                WriteElement collector el
```

Comment:
I notice that the C# VS tooling adds an empty newline between sections, which does help make them more visually distinct:

https://github.com/dotnet/roslyn/blob/8e4ab418a8f9703f7dfe3a66adc9b3876ef9382f/src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.DocCommentFormatter.cs#L130-L133

https://github.com/dotnet/roslyn/blob/8e4ab418a8f9703f7dfe3a66adc9b3876ef9382f/src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.DocCommentFormatter.cs#L152-L158

![image](https://github.com/dotnet/fsharp/assets/14795984/8c80c3c4-ebc4-4112-8d72-3fc1114eaf0c)


Should we do the same here?